src/xcode/ENA/ENA/Source/Services/TicketValidation/TVServiceIdentityDocumentProcessor.swift

```diff
@@ -0,0 +1,96 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+import ENASecurity
+
+protocol TVServiceIdentityDocumentProcessing {
```

Comment:
```suggestion
protocol TicketValidationServiceIdentityDocumentProcessing {
```
Using TV looks like old Objective C code to me

Reply:
Done